.github/workflows/plan-examples.yml

```diff
@@ -0,0 +1,68 @@
+name: plan-examples
+
+on:
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  getExampleDirectories:
+    name: Get example directories
+    runs-on: ubuntu-latest
+    # Skip running on forks since it won't have access to secrets
+    if: github.repository_owner == 'aws-ia'
```

Comment:
Instead of showing failures to users on forks, we can just skip the checks until we develop a new strategy for executing plan validations on all PRs (forks included)